### PR TITLE
SISRP-34288 - 9.2 Outage - CalCentral Maintenance Page

### PR DESCRIPTION
**Pull request by nkutub**
https://jira.berkeley.edu/browse/SISRP-34288

* change maintenace page for long 4 day outage.

### CODE REVIEW

File: public/maintenance/index_maintenance.html [hunk 1]
```diff
@@ -134,27 +134,34 @@
...
+         </div>
         </div>
       </div>
       <div class="footer">
         <div class="sprite" id="img_2_berkeley_logotype"></div>
       </div>
-    </div>
-  </body>
+		</div>
+ </body>
```
Comment (redconfetti) [inline public/maintenance/index_maintenance.html]: @nkutub - Why is the indentation on this so low quality?

Comment (nkutub) [inline public/maintenance/index_maintenance.html]: This is temp and only for this release, also was built by someone else not me

File: public/maintenance/index_maintenance.html [hunk 2]
```diff
@@ -134,27 +134,34 @@
...
-              <li><a href="https://bcourses.berkeley.edu">bCourses</a></li>
-              <li><a href="http://bconnected.berkeley.edu/">bConnected</a> (bMail, bCal, bDrive)</li>
-              <li><a href="https://www.berkeley.edu/">berkeley.edu</a></li>
-            </ul>
-          </p>
-          <p>Please <a href="http://ets.berkeley.edu/calcentral-feedback">contact us</a> for assistance with CalCentral.</p>
-        </div>
+              <li>Read more about the <a href="http://sis.berkeley.edu/9-2-upgrade">CalCentral Outage and Upgrade.</a></li>
+              <li>Search Summer Sessions class times and locations at <a href="http://classes.berkeley.edu/">Berkeley Academic Guide.</a></li>
+				<li>Access <a href="https://bcourses.berkeley.edu/">bCourses directly.</a></li>
```
Comment (lyttam) [inline public/maintenance/index_maintenance.html]: @nkutub I think the word 'directly' should be outside of the <a> tag, like this:
`<li>Access <a href="https://bcourses.berkeley.edu/">bCourses</a> directly.</li>`

File: public/maintenance/index_maintenance.html [hunk 3]
```diff
@@ -134,27 +134,34 @@
...
+				<li>Can’t find your class on the Berkeley Academic Guide? Contact <a href="http://studentcentral.berkeley.edu/">Cal Student Central</a> at <a href="http://www.berkeley.edu/map?sproulhall">120 Sproul Hall</a> for help.</li>
+              <li>Need additional help, email <a href="mailto:sishelp@berkeley.edu">SIS Support.</a></li>
```
Comment (lyttam) [inline public/maintenance/index_maintenance.html]: @nkutub the grammar police request that you add a question mark after "help", and then capitalize "Email":

`<li>Need additional help? Email <a href="mailto:sishelp@berkeley.edu">SIS Support.</a></li>`

**Pull request merged**